apps/vr-tests/src/stories/Menu.stories.tsx

```diff
@@ -0,0 +1,172 @@
+import { storiesOf } from '@storybook/react';
+import * as React from 'react';
+import Screener from 'screener-storybook/src/screener';
+import {
+  Menu,
+  MenuTrigger,
+  MenuPopover,
+  MenuList,
+  MenuItem,
+  MenuItemCheckbox,
+  MenuItemRadio,
+  MenuGroup,
+  MenuGroupHeader,
+  MenuDivider,
+} from '@fluentui/react-menu';
+import { Cut20Regular, Edit20Regular, ClipboardPaste20Regular } from '@fluentui/react-icons';
+import { FluentProviderDecorator } from '../utilities/index';
+
+storiesOf('Menu - basic', module)
```

Comment:
```suggestion
storiesOf('react-menu Menu - basic', module)
```

Can we use `react-menu` as prefix? Similarly to `react-button`:

https://github.com/microsoft/fluentui/blob/cb38c2453ce8de979b54c8cebc1b4763bf67d541/apps/vr-tests/src/stories/ReactButton.stories.tsx#L12